web/modules/custom/d_partners/d_partners.install

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use \Drupal\node\Entity\Node;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_install().
+ */
+function d_partners_install() {
+  // Create sample Partners content.
+  $partners = [
+    'Droptica' => [
+      'public://logo-droptica.svg' => 'https://droptica.pl',
+    ],
+    'Abventor' => [
+      'public://abventor-logotyp.svg' => 'https://www.abventor.com',
+    ],
+    'Famatech' => [
+      'public://famatech.svg' => 'https://www.famatech.pl',
+    ],
+    'Smartbees' => [
+      'public://smartbees_icon.svg' => 'https://smartbees.pl',
+    ],
+  ];
+
+  foreach ($partners as $title => $partner_details) {
+    foreach ($partner as $logoUri => $site) {
```

Comment:
Nie ma takiej zmiennej jak $partner, jest za to $partner_details ;)

Reply:
No i musimy się zdecydować - albo camelCase albo underscore

Reply:
fuckkk ;-)